src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        return userService.findByEmail(email).getPassword().equals(password);
```

Comment:
May cause null pointer exception in case userService.findByEmail returns null. 
```suggestion
        return userService.findByEmail(email).getPassword().equals(password);
```